pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -53,6 +54,9 @@ type stateData struct {
 	// phase for each node
 	// it's initialized in the PreFilter phase
 	podVolumesByNode map[string]*scheduling.PodVolumes
+	// guards map podVolumesByNode in the Filter phase, as the method may be called
+	// in parallel
+	podVolumesByNodeMutex sync.Mutex
```

Comment:
We can make it simpy:

```suggestion
	sync.Mutex
```

And then call `stateData.lock()`.